packages/state/src/hooks/beacon.ts

```diff
@@ -108,7 +107,7 @@ export const useAddPeer = () => {
       .deserialize(payload)
       .then(parsePeerInfo)
       .then(peer => WalletClient.addPeer(peer as ExtendedPeerInfo))
-      .then(refresh)
+      .then(() => refresh())
```

Comment:
why not just refresh?

Reply:
because it passes its first argument to the refresh function which also has an optional argument which has nothing to do with the peers